src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.4
 ---
 
+ * Add the ability to enable the profiler using a parameter
```

Comment:
`parameter` feels vague :)

Reply:
```suggestion
 * Add the ability to enable the profiler using a request parameter
```

Reply:
`a query parameter` ?

Reply:
But this works with all kind of "parameters", including attributes. I'll update this.